### DISCUSSION TITLE
Want to add metadata to Resource

### DISCUSSION THREAD

**Discussion by RalphSpangler** [ckan/ckan#6669] [General]
My team has created an extension that allows the addition of metadata to Resources via the API and UI. These additional metadata are global in nature which is good for what they are used for. 

I would like to add metadata that is specific to a Resource or group of Resources, but not all. This could be:

- per Resource
- per Dataset
- per Group
- per Organization
- per some other criteria such as Title

Some metadata would be required, some optional. Ability to query the metadata is required.

If this is already available in an extension, please let me know. If not, some suggestions would be good.

**Comment by tino097** [comment]
@RalphSpangler you could add the extension to https://extensions.ckan.org/
